components/director/cmd/tenantloader/README.md

```diff
@@ -0,0 +1,31 @@
+# Tenant loader
+
+## Overview
+
+It is an application that is being runned in `tenant-loader-job`. It's main task is to synchronize given tenants with Compass.
```

Comment:
IMO we should not mention where this is used, here it is a place for describing what this app is doing.